catroid/config/pmd.xml

```diff
@@ -50,12 +50,8 @@
         <exclude name="DefaultPackage" />
         <exclude name="UnnecessaryConstructor" />
         <exclude name="AvoidUsingVolatile" />
-        <exclude name="DoNotCallGarbageCollectionExplicitly">
-            <exclude-pattern>.*src\main\java\org\catrobat\catroid\ui\BaseActivity.java</exclude-pattern>
-        </exclude>
-        <exclude name="AvoidUsingNativeCode">
-            <exclude-pattern>.*CatroidApplication.java</exclude-pattern>
-        </exclude>
+        <exclude name="DoNotCallGarbageCollectionExplicitly" />
+        <exclude name="AvoidUsingNativeCode" />
```

Comment:
Since you're using `@SuppressWarnings`, you can remove following rules from the exclude lists entirely
* DoNotCallGarbageCollectionExplicitly
* AvoidUsingNativeCode
* TestClassWithoutTestCases